apps/src/lib/ui/accounts/ChangeEmailForm.jsx

```diff
@@ -154,5 +166,8 @@ export default class ChangeEmailForm extends React.Component {
 const styles = {
   input: {
     marginBottom: 4
+  },
+  radio: {
```

Comment:
Nit: can be simplified to `0px 5px`.

Reply:
That adds 5px of margin to the left side, which looks okay, but I was trying to keep it aligned. I don't have a great eye for this kind of thing though so open to thoughts!
<img width="681" alt="Screen Shot 2022-04-29 at 8 31 32 AM" src="https://user-images.githubusercontent.com/46464143/165976602-5445afdf-04cf-4444-9352-9b1ae59bb125.png">

Reply:
Oh you're right, I read that wrong originally. :woman_facepalming: